src/ripple/rpc/impl/RPCHelpers.h

```diff
@@ -241,7 +241,7 @@ extern beast::SemanticVersion const lastVersion;
 constexpr unsigned int apiInvalidVersion = 0;
 constexpr unsigned int apiVersionIfUnspecified = 1;
 constexpr unsigned int apiMinimumSupportedVersion = 1;
-constexpr unsigned int apiMaximumSupportedVersion = 1;
+constexpr unsigned int apiMaximumSupportedVersion = 2;
 constexpr unsigned int apiBetaVersion = 2;
```

Comment:
Don't change the `apiMaximumSupportedVersion` until the updated version is locked in and final and ready for release. We have `apiBetaVersion` variable right here and the `[beta_rpc_api]` config option available to allow testing the changes. (It might not be a bad idea to enable `BETA_RPC_API = true` by default in envconfig.cpp.)

Reply:
Just wondering what do you mean by "until the updated version is locked in and final and ready for release."? Do you you mean until PR #[4522](https://github.com/XRPLF/rippled/pull/4552) is merged into develop, so I don't have to change apiMaximumSupportedVersion to 2?

Reply:
Please see also #4573.

Reply:
Just to close the loop on this: API v2 will be considered "Done" in the 1.12 release. Any breaking changes in 1.13 will be API v3.